### ISSUE TITLE
Call to undefined function "acf_get_options_pages"

### ISSUE THREAD

**Issue by laurent-le-graverend** [gatsbyjs/wp-gatsby#214]
Facing this issue since the last update, it happens when saving a post.

```
PHP message: PHP Fatal error:
Uncaught Error: Call to undefined function WPGatsby\ActionMonitor\Monitors\acf_get_options_pages()
in /var/www/wordpress/wp-content/plugins/wp-gatsby/src/ActionMonitor/Monitors/AcfMonitor.php:40
```

Probably coming from #206.

**Comment by andy-dextrous** [comment]
Happening to me now too. Started within the last 2 weeks, not sure exactly when. It occurs every time I save any page or post that has ACF fields attached.

<img width="700" alt="Screen Shot 2022-02-14 at 5 07 12 pm" src="https://user-images.githubusercontent.com/76076968/153816093-e05afa9f-c9b6-4cfa-9cef-3c7603781c29.png">

**Comment by shawngrona** [comment]
Same for me, looks like issue goes away if I rollback to wp-gatsby 2.2.0

This issue surfaced for me when I tried saving the meta description in the Yoast plugin

**Comment by TylerBarnes** [comment]
This is fixed in version 2.3.1 (it's publishing now 😄)
Thanks for the bug report!
